### PR TITLE
Related Component Compare Window Update

### PR DESCRIPTION
**Pull request by masonisblue**
Formatted and filled out comparison window with live data
Started cleaning up code and console logs
Still have to implement live review star output
Still have a lot of refactoring to do, but feedback appreciated

### CODE REVIEW

File: client/src/components/Related/CompareWindow/index.jsx [hunk 1]
```diff
@@ -3,26 +3,24 @@ import { useState, useEffect } from "react";
...
-
-  };
-
-  return (
+    return (
     <>
       <ActionBtn onClick={toggleModal}>
         <FontAwesomeIcon icon={regular("star")} />
       </ActionBtn>
       {modal ?
```
Comment (markus-leben) [inline client/src/components/Related/CompareWindow/index.jsx:20]: It's always kinda weird how the modal windows just sort-of hang out near the things that spawn them in the DOM, isn't it? Makes doing those <></> wrappers necessary.

File: client/src/components/Related/Carousel/Card.jsx [hunk 2]
```diff
@@ -9,8 +9,6 @@ import { faStar } from '@fortawesome/free-solid-svg-icons';
 import { solid, thin, regular, brands, icon } from '@fortawesome/fontawesome-svg-core/import.macro';
 import whiteBackground from '/client/src/components/assets/images/whiteBackground.jpg';
 import outfit from '/client/src/components/assets/outfit.jsx'
-// import product from "../../../../../server/exampleData/product.json"
```
Comment (markus-leben) [inline client/src/components/Related/Carousel/Card.jsx:12]: Sorry I didn't fix these paths when I did my initial revision pass on the file pathing.

File: client/src/components/Related/index.jsx
[the remark this answers is not in the document]
Comment (markus-leben) [reply client/src/components/Related/index.jsx:1]: You should dedent starting with line 35 (the one declaring the starting id). Right now it looks like your whole thing is in useEffect, when that isn't the case.

**Review by markus-leben — CHANGES_REQUESTED** [review]

File: client/src/components/Related/Carousel/Card.jsx [hunk 3]
```diff
@@ -68,15 +66,16 @@ const StyledCard = ({item, mode, list, setList, product}) => {
   // console.log('itemlength', item.length);
```
[the remark this answers is not in the document]
Comment (masonisblue) [reply client/src/components/Related/Carousel/Card.jsx:66]: working on getting ratings in after compare modal window approved

File: client/src/components/Related/CompareWindow/index.jsx (hunk 1 above)
Comment (masonisblue) [reply client/src/components/Related/CompareWindow/index.jsx:20]: any changes necessary here

Comment (markus-leben) [reply client/src/components/Related/CompareWindow/index.jsx:20]: Nah this one's fine, just me griping about react.

**Review by markus-leben — APPROVED** [review]

**Comment by markus-leben** [comment]
Solid changes, let's ship it!

**Pull request merged**